api-reference-examples/go/threatexchange/threatexchange_test.go

```diff
@@ -3,12 +3,13 @@ package threatexchange
 
 import (
 	"github.com/stretchr/testify/assert"
+	"os"
 	"testing"
 )
 
-const (
-	appId     = "" // Fill this to make tests run
-	appSecret = "" // Fill this to make tests run
+var (
+	appId     = os.Getenv("TX_APP_ID")     // Set this for integration tests
```

Comment:
It looks like all these tests are actually integration tests requiring hitting the live api which has test data that can change. We should look at moving these to actual unit tests that don't rely on the live system.

Reply:
Agreed, right now how does this test working non-locally without these being set?

Reply:
Basically, all the tests below skip if these are not set (that was the behavior from before with the empty string), and the overall test run passes solely do to no failures.

So the current test runs in CI are basically just checks on if it compiles.